pkg/server/deploy/deploy.go

```diff
@@ -150,6 +152,12 @@ func (ops *DeployOperations) createOrUpdateDeploy(a *app.App, confFiles *DeployC
 			log.WithError(err).Errorf("Creating config to nginx of app %s", a.Name)
 			return
 		}
+	} else {
+		err := ops.k8s.DeleteConfigMap(a.Name, a.Name)
+		if !ops.k8s.IsNotFound(err) {
```

Comment:
It will return a nil error to the channel, right?

Reply:
It will return the error if it is not `not found`

Reply:
yes, you're right!

Reply:
Updated!